pom.xml

```diff
@@ -449,6 +449,7 @@
                 <module>sql-db/multiple-pus</module>
                 <module>sql-db/panache-flyway</module>
                 <module>sql-db/vertx-sql</module>
+                <module>sql-db/sql-reactive</module>
```

Comment:
I think that could be renamed to something like `sql-db/sql-reactive-panache`

Reply:
Module renamed